data/moves.ts

```diff
@@ -13714,7 +13714,7 @@ export const Moves: {[moveid: string]: MoveData} = {
 			}
 		},
 		onTryHit(target, source, move) {
-			this.add('-activate', target, 'move: Poltergeist', this.dex.getItem(target.item).name);
+			this.add('-item', target, this.dex.getItem(target.item), '[from] move: Poltergeist', '[of] ' + source);
```

Comment:
```suggestion
			this.add('-activate', target, this.dex.getItem(target.item).fullName, '[from] move: Poltergeist', '[of] ' + source);
```

Something like _this_ I think might be closer to what you want, though.

Reply:
Although the old version should in theory also work. Is there a reason you wanted to change it?

Reply:
no the server-side is useless, my bad